shared/common-adapters/input.desktop.js

```diff
@@ -199,7 +180,7 @@ class Input extends React.PureComponent<Props, State> {
               selection: newSelection,
             }
           })
-          this._onChangeText(this.getValue())
+          this._onChangeText(this._getValue())
```

Comment:
just noticed that we call `_transformText` here, even though we might be controlled. I guess in practice we don't use multiline with controlled components. I'll fix in a separate PR.